src/status_im/navigation/effects.cljs

```diff
@@ -8,10 +10,20 @@
     [taoensso.timbre :as log]
     [utils.re-frame :as rf]))
 
+(defn- set-status-bar-color
```

Comment:
maybe we should just keep this function "pure" and pass only theme in.

```
defn- set-status-bar-color
  [theme]
  (rn/set-status-bar-style
   (quo/theme-value
     "dark-content"
     "light-content"
     theme)
   true))
```
(rf/reg-fx :set-view-id-fx
 (fn [view-id]
   (rf/dispatch [:screens/on-will-focus view-id])
   (when-let [{:keys [on-focus options]} (get views/screens view-id)
   page-theme (:theme options)
   user-theme (quo.theme/get-theme)
   ]
     (set-status-bar-color (or page-theme user-theme))
     (when on-focus
       (rf/dispatch on-focus)))))
```